mlrun/model_monitoring/db/tsdb/tdengine/schemas.py

```diff
@@ -227,53 +233,68 @@ def _get_records_query(
 
 @dataclass
 class AppResultTable(TDEngineSchema):
-    def __init__(self, database: Optional[str] = None):
-        super_table = mm_schemas.TDEngineSuperTables.APP_RESULTS
+    def __init__(self, project: str, database: Optional[str] = None):
+        super_table = f"{mm_schemas.TDEngineSuperTables.APP_RESULTS}"
```

Comment:
Why f-string?
```suggestion
        super_table = mm_schemas.TDEngineSuperTables.APP_RESULTS
```

Same for the other tables.

Reply:
oh just a leftover from a change that wasn't implemented